lib/Controller/DocumentController.php

```diff
@@ -1084,7 +1085,7 @@ private function putRelative($fileId, $owner, $editor, $suggested) {
 		$serverHost = $this->request->getServerProtocol() . '://' . $this->request->getServerHost();
 
 		// Continue editing
-		$attributes = WOPI::ATTR_CAN_VIEW | WOPI::ATTR_CAN_UPDATE | WOPI::ATTR_CAN_DOWNLOAD | WOPI::ATTR_CAN_PRINT;
+		$attributes = WOPI::ATTR_CAN_VIEW | WOPI::ATTR_CAN_UPDATE | WOPI::ATTR_CAN_PRINT;
```

Comment:
`WOPI::ATTR_CAN_DOWNLOAD` is undefined and not so different than `WOPI::ATTR_CAN_VIEW` IMO

Reply:
I raised issue #335